src/css/protocol/components/article.scss

```diff
@@ -4,3 +4,13 @@
 
 @use '~@mozilla-protocol/core/protocol/css/includes/lib' as * with ($image-path: '/static/protocol/img');
 @import '~@mozilla-protocol/core/protocol/css/components/article';
+
+.mzp-c-article {
+    margin: 0 auto;
+    max-width: $content-lg;
```

Comment:
This is actually a bit too wide, and makes for a less than optimal line length for readability. If we set it to `$content-md` it's more readable, but would change current instances of articles on future.m.o... if we're OK with making that change I'd like to do it, but didn't take the liberty just yet. It doesn't breaking anything and does look better, and is how it should have been all along, but it will change alignments with other component on the page. So not sure how to proceed...

Reply:
Can we hack some CSS on actual live pages via devtools to see what the 'damage' is like?